scripts/spack/configs/other/tempus52/spack.yaml

```diff
@@ -163,12 +165,8 @@ spack:
       buildable: false
     llvm:
       externals:
-      - spec: llvm@14.0.6+clang~lld~lldb
+      - spec: llvm@19.1.1+clang~lld~lldb
```

Comment:
when is it appropriate to add `extra_attributes` to this section? i noticed its in the ubuntu one but not tempus and toss4

```yaml
        extra_attributes:
          compilers:
            c: /usr/bin/clang
            cxx: /usr/bin/clang++
```

Reply:
oh is it just for `llvm-doe`?